bin/notify.yaml

```diff
@@ -0,0 +1,2 @@
+teams:
+  all: iarspider, smuzaffar, aandvalenzuela
```

Comment:
why not `all: cms-sw/externals-l2` ? This way we only need to update the the users list vai https://github.com/cms-sw/cms-bot/blob/master/categories.py

Reply:
I think by default `cms-sw/externals-l2` should get all notifications